### ISSUE TITLE
[mickmek] deepcognition.ai soda bottle competition (18h)

### ISSUE THREAD

**Issue by shadiakiki1986** [teamshadi/ssoc18#6]
@MickMek here's your issue to coordinate about deepcognition.ai model

- [x] warm up (3h)
    - try out deepcognition.ai
- [x] first model version including pre-trained VGG (5h)
    - achieved 18% accuracy on validation set
- [x] model version 2 (5h)
    - move cropping earlier
    - reduce the image sizes to have faster computation
    - [x] submission score: 34%
    - epochs = 10, batch size = 32, 1 hour of training time, train/validation/test: 80%/10%/10%
    - [x] export keras code from deepcognition and upload on server 
- [x] model version 3 (5h)
    - thoughts to improve the model:
        - simplify the model
        - train on full dataset for the submission
        - either fix or remove cropping (current setup just crops boundary 1 pixel)
        - circular convolution matrix
    - [x] submission score: 65% (rank: 36 as of 2018-07-09)

**Comment by MickMek** [comment]
Update:
Completed model2 with around 30% testing accuracy (cropping moved and some modifications to the CNN topology). Submitted the second version of the model to the deepcognition site.

**Comment by MickMek** [comment]
Update:
Model version 3 achieved 99% training accuracy and 82% validation accuracy (training for 10 epochs).
Next step is to reduce overfitting with batch_norms and dropouts, before testing "deeper" networks.
Submitted the third version of the model to the deep cognition site (still no results for the 3 submitted models)

**Comment by shadiakiki1986** [comment]
I think that since we did not receive any submission result by email, send
them a message via their contact button asking them why you're not getting
any results by email.

**Comment by shadiakiki1986** [comment]
@MickMek updated issue as discussed over the phone

**Comment by MickMek** [comment]
Update:
Improved model (optimized parameters and topology) to attain 97% validation accuracy, and scores 62.5% on deep cognition. 
The code is now on the server

**Comment by shadiakiki1986** [comment]
Wow! I see you're number 36 on the leader board. Well done
